### PR TITLE
Migarting travis jobs to Semaphore

### PR DESCRIPTION
**Pull request by PrasanthV454**
Migrating current travis jobs to semaphore.

Flake8, docs and integration tests will be executed every time but only on linux x64 machines.

Unit Tests will be executed on all machine types everytime.

### CODE REVIEW

File: .semaphore/semaphore.yml [hunk 1]
```diff
@@ -51,51 +51,151 @@ blocks:
...
       env_vars:
         - name: OS_NAME
           value: linux
         - name: ARCH
-          value: arm64
+          value: x64
       jobs:
         - name: Build
           commands:
             - ./tools/build-manylinux.sh "${LIBRDKAFKA_VERSION#v}"
```
Comment (anchitj) [inline .semaphore/semaphore.yml:89]: We can probably use `tools/wheels/build-wheels.sh` script here for keeping the compatibility with the Travis job.

Comment (PrasanthV454) [reply .semaphore/semaphore.yml]: Done

**Review by anchitj — APPROVED** [review]

**Review by emasab — CHANGES_REQUESTED** [review]
It seems in Linux x64 flake8 is being run on the whole _venv, causing a huge [log](https://storage.googleapis.com/d9mnuahu5ecvy9utyjuyainyog83mh/artifacts/jobs/04b9bc4d-eea4-4922-9ad0-ddc3a163c14f/agent/job_logs.txt?Expires=1676460227&GoogleAccessId=artifacthub%40semaphore2-prod-artifacts.iam.gserviceaccount.com&Signature=bw6BCD5c8xaEq7EpDO4bpn4AI4xEBAMWST4Ehqk8vhRK6RdfHZ7DLM5qNw3xnBuUYbxPG7aJI%2F81Xpv8pZYHCK%2B2czjBDv8ggSqSXsE0am%2FOsFyJj1BlB5V5ZHBQa7Qax6PF8m2kAgLL6PyDvQ3iq%2FagaAxBI9TI2%2BrM42mFqd%2BbSmogLBh3QHPZdAxM2ezHqV7f2U9tEZpQGFnmt3rylE%2B71vzxHU5HlchofQrgxU9Ll90p4t8BNLrRF1bLz%2FOHZ0Zxnp2Bl8o92WZYBCZPQ35ksKXSY1BAAeATYowpM9ZyRmyINsHfw4JDGdZzy%2BZQKfKQTCMfC6A244D%2FIXmD9w%3D%3D&response-content-disposition=inline&response-content-type=text/plain;%20charset=utf-8)

**Comment by emasab** [comment]
Is there a test run for wheel generation?

**Comment by PrasanthV454** [comment]
> Is there a test run for wheel generation?
Yes. CIBW_TEST_COMMAND is being set before building the wheels.

**Comment by PrasanthV454** [comment]
> It seems in Linux x64 flake8 is being run on the whole _venv, causing a huge [log](https://storage.googleapis.com/d9mnuahu5ecvy9utyjuyainyog83mh/artifacts/jobs/04b9bc4d-eea4-4922-9ad0-ddc3a163c14f/agent/job_logs.txt?Expires=1676460227&GoogleAccessId=artifacthub%40semaphore2-prod-artifacts.iam.gserviceaccount.com&Signature=bw6BCD5c8xaEq7EpDO4bpn4AI4xEBAMWST4Ehqk8vhRK6RdfHZ7DLM5qNw3xnBuUYbxPG7aJI%2F81Xpv8pZYHCK%2B2czjBDv8ggSqSXsE0am%2FOsFyJj1BlB5V5ZHBQa7Qax6PF8m2kAgLL6PyDvQ3iq%2FagaAxBI9TI2%2BrM42mFqd%2BbSmogLBh3QHPZdAxM2ezHqV7f2U9tEZpQGFnmt3rylE%2B71vzxHU5HlchofQrgxU9Ll90p4t8BNLrRF1bLz%2FOHZ0Zxnp2Bl8o92WZYBCZPQ35ksKXSY1BAAeATYowpM9ZyRmyINsHfw4JDGdZzy%2BZQKfKQTCMfC6A244D%2FIXmD9w%3D%3D&response-content-disposition=inline&response-content-type=text/plain;%20charset=utf-8)

we exclude the venv directory while running flake8

**Comment by emasab** [comment]
I was checking artifact generation with a test tag and it's [failing](https://confluentinc.semaphoreci.com/jobs/cae4f461-339d-43c8-b2d8-f6f3d31c325c) on windows. It can be that the windows agent has been modified and hasn't got python anymore. @PrasanthV454 please check this.

**Review by emasab — APPROVED** [review]
Great job @PrasanthV454, thanks for the last improvements!

**Pull request merged**